packages/block-editor/src/components/block-list/style.scss

```diff
@@ -1024,3 +994,41 @@
 		margin-bottom: auto;
 	}
 }
+
+.components-popover.block-editor-block-list__block__popover {
+	z-index: 29;
+}
+
+.components-popover.block-editor-block-list__block__popover .components-popover__content {
+	min-width: auto;
+	background: none;
+	border: none;
+	box-shadow: none;
+	overflow-y: visible;
+	// Allow clicks through the wrapper for the block movers, which may be
+	// positioned next to the toolbar.
+	pointer-events: none;
+
+	> * {
+		pointer-events: all;
+	}
+}
+
+.components-popover.block-editor-block-list__block__popover[data-x-axis="right"] .components-popover__content {
+	margin-left: 0 !important;
+}
+
+.components-popover .block-editor-block-contextual-toolbar,
+.components-popover .block-editor-block-list__breadcrumb {
+	margin-bottom: $block-padding - $border-width;
+	margin-left: - $block-padding;
+}
+
+// Make space for the block movers. Ideally they should move within the toolbar.
+.components-popover .block-editor-block-contextual-toolbar[data-align="wide"] {
+	margin-left: 86px;
+}
+
+.components-popover .block-editor-block-contextual-toolbar[data-align="full"] {
+	margin-left: 86 + 2 * $block-padding;
+}
```

Comment:
Do we relaly need all the specificity in these rules. Can't we just replace `components-popover` with `block-editor-block-list__block__popover`.

Also that className don't follow our guidelines, it should be just `block-editor-block-list__block-popover`

Reply:
Thanks, I'll adjust it.